homeassistant/components/whirlpool/const.py

```diff
@@ -5,12 +5,12 @@
 DOMAIN = "whirlpool"
 CONF_BRAND = "brand"
 
-CONF_REGIONS_MAP = {
+REGIONS_CONF_MAP = {
```

Comment:
```suggestion
REGIONS_MAP = {
```
I would just skip conf altogether, as in, it's not strictly tied to CONF, it just so happens to be like that